src/util.ts

```diff
@@ -357,3 +357,43 @@ export const toCamelCase = (str: string) =>
 
 export const parseBool = (v: string): boolean | string =>
   ({ true: true, false: false })[v] ?? v
+
+// prettier-ignore
+export const parseEnvFromFile = (
+    content: string,
+  ): NodeJS.ProcessEnv => {
+    return content
+      .split(/\r?\n/)
+      .reduce<NodeJS.ProcessEnv>((acc, line) => {
+        const cleanedLine = line.replace(/[\r?\n]+/g, '').trim();
```

Comment:
I believe it could be much shorter:
```js
const [k, v] = line.trim()split('=', 1)
if (k & v) m[k] = v
return m
```

Reply:
It won't work that way. 
Examples
```js
split('=', 1);/// == [key] 
split('=', 2) /// [key, value] 

```
But not works if string contains more then 1 symbol '='.

Possible solutions

```js
const r = line.trim().match(/^([^=]+)=(.*)$/);
if (r != null) acc[r[0]] = r[1];
return acc;
```

or 
```
const [k, ...v] = line.trim().split("=");
if (k && v.length) acc[k.trim()] = v.join("=").trim();
return acc;
```

Reply:
```js
const [k,v] = line.split(/(?<!=)=/)
```

Reply:
Why splitting on a = will not work?

Reply:
```js
'a=======b'.split('=')
[
  'a', '', '', '',
  '',  '', '', 'b'
]
```

Reply:
d1

Reply:
Split with limit: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/split#limit

Reply:
I use this in first iteration. 

```js
const [k, v] = 'fdfd=re=rer'.trim().split('=', 2) 
k = 'fdfd'
v = 're'
```

If we have use `k = re=rer`

```js
const [k, ...v] = 'fdfd=re=rer'.trim().split('=') 
k = 'fdfd'
v.join('=') = re=rer
```

Reply:
I described the behavior #discussion_r1895896418

Reply:
```js
const line = 'fdfd=re=rer'
const [key] = line.split('=', 1)
const value = line.slice(key.length+1)

```

Reply:
The first iteration used this approach, it looks like I'll bring it back.

Reply:
d1